source/Renderer/shaders/displaymapping.glsl

```diff
@@ -0,0 +1,53 @@
+// KHR_displaymapping_pq
+
+uniform float u_ApertureFactor;  // Calculated by using max light intensity value of the scene
+
+const float maxComponent = 10000.0;
+const bool displayIsSDR = true;
+
+// Reference PQ OOTF of ITU BT.2100: https://www.itu.int/rec/R-REC-BT.2100/en
+vec3 BT_2100_OOTF(vec3 color, float rangeExponent, float gamma) 
+{  
+    vec3 nonlinear = 1.099 * pow(rangeExponent * color, vec3(0.45)) - 0.099;  
+    return 100.0 * pow(nonlinear, vec3(gamma));
+}  
+
+
+vec3 OOTF(vec3 apertureAjustedColor)
+{
+    float rangeExponent;
+
+    if(displayIsSDR)
```

Comment:
Since this value will not change for the duration of the current swapchain I would advice setting it as a Uniform (to avoid having an `if` statepent per fragment.